server/src/crm/xml/xml.service.ts

```diff
@@ -2,7 +2,6 @@ import { Injectable } from '@nestjs/common';
 import { CrmService } from '../crm.service';
 import * as request from 'superagent'
 import { ConfigService } from '../../config/config.service';
-import { ADAL } from '../../_utils/adal';
```

Comment:
Can we delete the xml.service.ts altogether?

Reply:
nope =/